examples/linear_model/plot_huber_vs_ridge.py

```diff
@@ -0,0 +1,66 @@
+"""
+=======================================================
+HuberRegressor vs Ridge on dataset with strong outliers
+=======================================================
+
+Fit Ridge and HuberRegressor on a dataset with outliers.
+
+The example shows that the coefficients in ridge are strongly influenced
+by the outliers are added to the dataset. The Huber regressor is less
```

Comment:
I would use "present" rather than added, as no-one adds outliers intentionally ;).
The sentence currently doesn't parse. Either "when outliers are present" or "by the outliers that are present".